src/ui/public/courier/search_strategy/default_search_strategy.js

```diff
@@ -68,7 +68,15 @@ export const defaultSearchStrategy = {
       failedSearchRequests,
     } = await serializeAllFetchParams(allFetchParams, searchRequests, serializeFetchParams);
 
-    const searching = es.msearch({ body: serializedFetchParams });
+    const msearchParams = {
+      body: serializedFetchParams,
+    };
+
+    if (maxConcurrentShardRequests !== 0) {
+      msearchParams.max_concurrent_shard_requests = maxConcurrentShardRequests;
```

Comment:
While this technically works, I'd like to change it to use the snake case version once the es.js API generation script is re-run to make `maxConcurrentShardRequests` a first class citizen.